cppcache/src/Pool.cpp

```diff
@@ -45,6 +45,9 @@ std::chrono::milliseconds Pool::getReadTimeout() const {
   return m_attrs->getReadTimeout();
 }
 
+std::string Pool::getSniProxyHost() const { return m_attrs->getSniProxyHost(); }
```

Comment:
formatting looks off here...

Reply:
Formatting is consistent with our 80 column limit and other single line functions in this file.

Reply:
okay, github just wrapped it, which made it look bad